datadog_checks_dev/datadog_checks/dev/tooling/templates/docs/setup/configuration/log_collection.yaml

```diff
@@ -0,0 +1,28 @@
+name: Log collection
+header_level: 4
+description: |
+  _Available for Agent versions >6.0_
+
+  1. Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file:
+
+    ```yaml
+    logs_enabled: true
+    ```
+
+  2. Add this configuration block to your `{check_conf_path}` file to start collecting your {display_name} logs:
+
+    {check_log_conf_snippet}
```

Comment:
```suggestion
    `{check_log_conf_snippet}`
```

Reply:
This block will tend to span multiple lines, see here for example: https://github.com/DataDog/integrations-core/pull/7891/files#diff-d5f9fee76763bafcbfc3d64f5984d309607ee02a9678088d8eaacc1bddec9999R61